aws-alb/docs/ELBAuthError.md

```diff
@@ -0,0 +1,8 @@
+---
+title: ELBAuthError
+brief: The number of user authentications that could not be completed because an authenticate action was misconfigured, the load balancer couldn't establish a connection with the IdP, or the load balancer couldn't complete the authentication flow due to an internal error.
+metric_type:
```

Comment:
Why is metric_type always blank?

Reply:
As I learned AWS metrics can have more than one metric type! They are differentiated based on a dimension called stat. It's apparently a specialized case for AWS metrics. The UI doesn't use the value. I think it picks whatever is the first metric type it sees for that metric even though apparently that's not quite accurate for AWS.

Reply:
Ah, right, I remember now. I'd like to ask @beccatortell if she would like to have anything in this field, instead of leaving it blank. She just got back from PTO so am not sure when she might have time to look at this, but I'll drop her a line in slack.